.editorconfig

```diff
@@ -0,0 +1,4 @@
+[*.{.js,.json}]
```

Comment:
I thought of adding this file to help contributors respect the coding conventions